tools/travis/build.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
+set -e
+
 # Build script for Travis-CI.
 
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
-WHISKDIR="$ROOTDIR/openwhisk"
+WHISKDIR="$ROOTDIR/../openwhisk"
```

Comment:
this will not work unless you change the location of where you cloned the openwhisk dir, which you have not done